### DISCUSSION TITLE
ConfirmEmail override not working in prod

### DISCUSSION THREAD

**Discussion by jaypeemayo** [OrchardCMS/OrchardCore#10985] [Q&A]
I override ConfirmEmail and TemplateUserLostPassword but it only works locally. When I deployed it to production it is still using the default view. did anyone encounter this problem?

![image](https://user-images.githubusercontent.com/21273774/148633881-ed3e810d-e162-4072-9a1b-bc096e55cc1d.png)

**Comment by Piedone** [comment]
Is this a duplicate of https://github.com/OrchardCMS/OrchardCore/discussions/11013?
